packages/fgtclb/typo3-category-types/Classes/ViewHelpers/Be/CategoryViewHelper.php

```diff
@@ -59,7 +59,7 @@ public static function renderStatic(
 
         /** @var CategoryRepository $repository */
         $repository = GeneralUtility::makeInstance(CategoryRepository::class);
-        $categories = $repository->findByGroupAndPageId($arguments['group'], $arguments['page']);
+        $categories = $repository->findByGroupAndPageId($arguments['group'], $arguments['page'], true);
```

Comment:
See above - Context should be passed/set, eventually a cloned one with enforced `includeHiddenContent = true` visibility aspect setting.